src/qw/md.py

```diff
@@ -28,3 +31,292 @@ def text_under_heading(text: str, heading: str) -> str:
         raise QwError(msg)
 
     return "\n".join(sub_heading_lines).strip()
+
+
+# Breaking Markdown text into paragraphs of some type
+FENCED_RE = re.compile(r"(?:^|\n+)(?:~{3,}|`{3,})\s*(.*?)\n([\S\s]*?)\n(?:~{3,}|`{3,})(?:$|\n+)")
```

Comment:
oopf, hello regex

Reply:
Yeah...
Should I be using that weird extended syntax with whitespace and comments?

Reply:
I think probably fine for now, takes a little while to get your head in, but does make sense. Could comment a bit about the pattern that you're using